### PR TITLE
Update entrypoint.md to add consistent CLI syntax

### PR DESCRIPTION
**Pull request by nstankov-bg**
### What does this PR do?
- Makes sure CLI syntax is consistent. (one per row)

| Before | After 
|--------|--------|
|    | 

### Motivation

I was being lazy and could not easily copy-paste for my compose file.

### More

- [X] Added/updated documentation

### CODE REVIEW

**Comment by rtribotte** [comment]
Hello @nstankov-bg,

Thanks for your contribution!

We took a look at the PR and we think that the current syntax/form of the CLI examples in the documentation is preferable.
We think it is simpler to break it down into multiple lines than inlining the arguments.

As a side note, if we were inclined to change it, for the sake of consistency, we would have to change every example in the documentation.

Unfortunately, for those reasons, we are declining your PR.

File: docs/content/routing/entrypoints.md [hunk 1]
```diff
@@ -288,7 +290,9 @@ It can be used to override the authority in the `alt-svc` header, for example if
     ```
     
     ```bash tab="CLI"
-    --experimental.http3=true --entrypoints.name.http3.advertisedport=443
+    ## Static configuration
```
Comment (rtribotte) [inline docs/content/routing/entrypoints.md:293]:
```suggestion
```

File: docs/content/routing/entrypoints.md [hunk 2]
```diff
@@ -258,7 +258,9 @@ In most scenarios, this entryPoint is the same as the one used for TLS traffic.
     ```
     
     ```bash tab="CLI"
-    --experimental.http3=true --entrypoints.name.http3
+    ## Static configuration
```
Comment (rtribotte) [inline docs/content/routing/entrypoints.md:261]:
```suggestion
```

**Review by rtribotte — CHANGES_REQUESTED** [review]

**Review by rtribotte — APPROVED** [review]
Thanks 👍

**Comment by nstankov-bg** [comment]
Thank you!

**Pull request merged**